### DISCUSSION TITLE
When to hit start on qdomyos for Peloton?

### DISCUSSION THREAD

**Discussion by tuffdadsf** [cagnulein/qdomyos-zwift#667] [Q&A]
Am I supposed to hit start on qdomyos when I start the Peloton workout or wait until the intro is done and the workout really begins? I am trying to sync the trainer resistance with my iFit bike and it's always off. Sometimes by a few seconds and sometimes longer. It does change to the correct thing eventually it's just off timing wise.

I see there is a tile I can activate to do a Peloton Offset. Not sure which direction (plus or minus) I need to put it so it can catch up. If the video is ahead of the resistance script do I hit plus?

BTW - the program is great and I really appreciate how well it works in general. Thanks!

**Comment by cagnulein** [comment]
Hi @tuffdadsf be sure to hit "yes" when the timer on the peloton app is "0:00"
Also, since the peloton instructor metrics aren't precise at all (even for peloton bike owners!) you need to trim the offset with the peloton offset tile.
Let me know

**Reply by tuffdadsf** [reply]
Thanks for the reply!

As for the offset, if the call from the instructor takes 5 seconds for the bike to react - I would set the offset to +5?

**Reply by cagnulein** [reply]
the plus one, you can double check it using the time to next tile

**Reply by tuffdadsf** [reply]
"time to next"? I see tiles for time and remaining time/row and moving time.

**Reply by cagnulein** [reply]
yes that one :)

**Reply by tuffdadsf** [reply]
I'll just test them all tonight. Thanks again for the help!

**Reply by cagnulein** [reply]
let me know ;)
